server/views/results.html

```diff
@@ -237,7 +237,7 @@ <h2 class="govuk-summary-card__title">
                 Decide what you need for your planning application
               </dt>
               <dd class="govuk-summary-list__value">
-                <a href="#">I need help deciding what to include in my planning application.</a>
+                <a href="next-steps?polygon={{polygon}}">I need help deciding what to include in my planning application.</a>
```

Comment:
Should the href have a leading `/` included, i would myself? I know this works fine, but technically if the result page's route changed up a level, and became eg `host/dir/results` then because there's no leading slash it would try to resolve to `host/dir/next-steps`

Reply:
good point